src/sentry/replays/post_process.py

```diff
@@ -24,35 +24,49 @@ def generate_normalized_output(
 ) -> Generator[None, None, Dict[str, Any]]:
     """For each payload in the response strip "agg_" prefixes."""
     for item in response:
-        item["id"] = item.pop("replay_id")
+        item["id"] = item.pop("replay_id", None)
```

Comment:
just a thought but i wonder if we could use the query config field_alias as the keys for item. messy thing would be building the nested return values